gridstatus/eia.py

```diff
@@ -1,12 +1,17 @@
 import concurrent.futures
+import datetime
 import json
 import os
+import re
+from urllib.request import urlopen
```

Comment:
can we switch this to using the `requests` library? this is more consistent with the rest of the library. 

see example here: https://github.com/kmax12/gridstatus/blob/main/gridstatus/ercot.py#L1422